site/content/en/docs/administration/basics/installation.md

```diff
@@ -76,7 +76,12 @@ For access from China, read [sources for users from China](#sources-for-users-fr
   git clone https://github.com/opencv/cvat
   cd cvat
   ```
-
+- To access CVAT over a network or through a different system, export `CVAT_HOST` environment variable
```

Comment:
From my point of view, it is better not to duplicate this, but add a link to the corresponding section of the FAQ page: https://openvinotoolkit.github.io/cvat/docs/faq/#how-to-change-default-cvat-hostname-or-port

Reply:
FAQ and documentation has different purpose. Let's have the duplication for now. In the future we will add more details into documentation but FAQ should be short.